src/posts/how-to-hide-content.md

```diff
@@ -53,6 +53,8 @@ With modern browsers and IE9 and up, the visually hidden selector can be written
 
 This will ensure that if an interactive element receives focus, the styles of the .`visually-hidden` class will be undone and the focusable content will be exposed.
 
+The CSS class `.visually-hidden` is also commonly known as `.sr-only`, where sr stands for "screen readers".
```

Comment:
I think we shy away from the idea that "accessibility is for screen readers only" and that's why we're using `.visually-hidden`. But I think a note about `.sr-only` classes existing in popular in frameworks is good to point out. 

```suggestion
Many popular CSS frameworks provide a `.visually-hidden` or `.sr-only` class, where `sr` stands for "screen readers".
```

Reply:
agreed.  screen readers are merely one type of assistive technology that visually hidden content can leverage or be hindered by.    there was an effort to try to move people away from using/saying "sr-only".  but agreed it is worth mentioning it so at least to inform people they're the same thing.

Reply:
I understand that effort and the intentions behind it. 

May I confess that as a non-native English speaker, I struggle to write "visually-hidden" compared to sr-only. Also, I've seen people confusing it with CSS `visibility: hidden` because of its similar name.
- visibility-hidden -> more inclusive
- sr-only -> easier to use

With this said, do you know of any other possible naming (even if not common) for this technique that could be both easy and inclusive?

Reply:
i've seen 'at-only' as another option.  i don't know how well that caught on though... i wouldn't necessarily mention it.  but yah, naming things is hard, particularly when one needs to be both descriptive with the name, while concise and aware of everyone's level of understanding/comfort with the name.   :: giant everlasting shrug ::

Reply:
Interesting name but agree that it shouldn’t be mentioned. Thanks @scottaohara , I have nothing else to say here